concepticondata/conceptlists/KBA-2021-912.tsv

```diff
@@ -0,0 +1,913 @@
+KBA-2021-912-767	767	m597-k-bp-fl-fa	to think (believe, be of the opinion)	1513	THINK (BELIEVE)		penser, croire	denken, glauben, meinen	dink, glo, meen			8. Sensations and Feelings			
+KBA-2021-912-768	768	m598-k-bp80-fl-fa	to see (experience)	1409	SEE		voir	sehen	sien	go bona		8. Sensations and Feelings	M556	s130	lj089
+KBA-2021-912-769	769	m599-k-bp-fl-fa	to look at (activity)	1819	LOOK		regarder	ansehen, betrachten, anschauen	aankyk	go leba		8. Sensations and Feelings	M154		
+KBA-2021-912-770	770	m6-k99-bp-fl-fa	girl	1646	GIRL		fille	Mädchen	meisie	morweetsana, ba-; mosetsana, ba-		1. Social life  1A.Social structure	M127		
+KBA-2021-912-771	771	m60-k-bp-fl-fa	quarrel	1848	QUARREL		lutte, querelle	Streit, Zank	stryd, twis	dikomana,komano		1C. Social and Antisocial Behaviour	M160		
+KBA-2021-912-772	772	m600-k-bp-fl-fa	to look   (phenomenon)	1819	LOOK		apparaître, ressembler	aussehen				8. Sensations and Feelings			
+KBA-2021-912-773	773	m601-k-bp82-fl-fa	to hear  (experience)	1408	HEAR		entendre	hören	hoor	go utlwa		8. Sensations and Feelings	M554	s069	lj061
+KBA-2021-912-774	774	m602-k-bp83-fl-fa	to listen to (activity)	1608	LISTEN		écouter	anhören, sich anhören, zuhören	luister			8. Sensations and Feelings			
+KBA-2021-912-775	775	m603-k-bp-fl-fa	to sound (phenomenon)												
+KBA-2021-912-776	776	m604-k-bp-fl-fa	sound	1811	SOUND OR NOISE		son	Geräusch	geluid, klank	modumo ('noise'?)		8. Sensations and Feelings	M555		
+KBA-2021-912-777	777	m605-k-bp-fl-fa	noise	1182	NOISE		bruit	Lärm	geraas, lawaai			8. Sensations and Feelings			
+KBA-2021-912-778	778	m606-k-bp-fl-fa	to feel (experience)	1079	FEEL		sentir	fühlen	voel	go utlwa		8. Sensations and Feelings	M557		
+KBA-2021-912-779	779	m607-k-bp-fl-fa	to touch, feel (activity)	2559	FEEL (TACTUALLY)		toucher	anfassen, befühlen, betasten	aanraak	go tshwara		8. Sensations and Feelings	M557		
+KBA-2021-912-780	780	m608-k-bp-fl-fa	to feel (phenomenon)												
+KBA-2021-912-781	781	m609-k-bp-fl-fa	to taste  (experience, involuntarily)				goûter	schmecken				8. Sensations and Feelings			
+KBA-2021-912-782	782	m61-k-bp-fl-fa	to strike (hit)	353	STRIKE		frapper, donner un coup	schlagen, hauen	stoot, slaan	go itaya		1C. Social and Antisocial Behaviour	M164	s073	lj036
+KBA-2021-912-783	783	m610-k-bp-fl-fa	to taste (activity)	21	TASTE (SOMETHING)		goûter	probieren, schmecken, abschmecken	smaak, proe	go utlwa		8. Sensations and Feelings	M558		
+KBA-2021-912-784	784	m612-k-bp-fl-fa	sweet	717	SWEET		sucré, doux	süß	soet	monate		8. Sensations and Feelings	M594		lj089
+KBA-2021-912-785	785	m613-k-bp-fl-fa	bitter	887	BITTER		amer,âpre	bitter	bitter	botsarara		8. Sensations and Feelings	M559		lj028
+KBA-2021-912-786	786	m614-k-bp81-fl-fa	to smell (experience, involuntarily)			SMELL (PERCEIVE)	sentir (renifler; sentir)	riechen (riechen an; riechen)	ruik (ruik aan)			8. Sensations and Feelings			
+KBA-2021-912-787	787	m615-k-bp-fl-fa	to smell (activity)	1586	SMELL (PERCEIVE)	SNIFF									
```

Comment:
Is the "SNIFF" a translation here?

Reply:
I think it should rather be unmapped then as we don't currently have a concept for smelling in the sense of active perception ("to smell a flower" compared to "he smelled smoke").